R/caretEnsemble.R

```diff
@@ -151,11 +159,23 @@ predict.caretEnsemble <- function(object, keepNA = TRUE, se = FALSE, return_weig
     conf <- sweep(conf, MARGIN=2, object$weights,`*`)
     conf <- apply(conf, 1, function(x) x / sum(x, na.rm=TRUE))
     conf <- t(conf); conf[is.na(conf)] <- 0
-    est <- apply(preds, 1, function(x){
-      weighted.mean(x, w=object$weights, na.rm = TRUE)
-    })
-    se.tmp <- apply(preds, 1, FUN = wtd.sd, weights = object$weights,
-                    normwt = TRUE, na.rm = TRUE)
+
+    if(dim(preds)[1]<1000){ # use apply for small data
```

Comment:
nitpick: put your comments on their own line.  In this case "# use apply for small data" should come before the code it refers to.